pkgs/tools/filesystems/e2fsprogs/default.nix

```diff
@@ -14,6 +14,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig texinfo ];
   buildInputs = [ libuuid gettext ];
 
+  # Only use glibc's __GNUC_PREREQ(X,Y) (checks if compiler is gcc version >= X.Y) when using glibc
+  # It's safe to use on non-gcc is glibc's features.h is being used.
```

Comment:
It is not clear to me what you mean with:

```
# It's safe to use on non-gcc is glibc's features.h is being used. 
```

Reply:
Typo "is" -> "if"; dropping that line since not really useful.

What I was intending to say was that this is glibc-specific, not (glibc+gcc)-specific, despite checking for a gcc version number.  At least it will be defined to something, instead of failing to compile.

For "special reasons" clang defines macros claiming to be GCC 4.2, so this is all a bit confusing anyway :grin:.